model/common/tests/common/decomposition/mpi_tests/test_mpi_decomposition.py

```diff
@@ -329,3 +333,20 @@ def test_halo_exchange_for_sparse_field(
     exchange.exchange_and_wait(dims.EdgeDim, c_lin_e_field)
 
     assert test_helpers.dallclose(c_lin_e_field.asnumpy(), c_lin_e_ref.asnumpy())
+
+
+@pytest.mark.mpi
+@pytest.mark.parametrize("processor_props", [True], indirect=True)
+def test_exchange_on_dummy_reduction(
+    processor_props: definitions.ProcessProperties,
+) -> None:
+    my_rank = processor_props.rank
+    print(f"rank={my_rank}")
+
+    global_reduc = definitions.create_global_reduction(processor_props)
+    arr = np.arange(3) - my_rank
+
+    min_val = global_reduc.min(arr)
+    expected_val = -my_rank
```

Comment:
That will fail: The expected value is a constant, the same on all nodes because you do a reduction over all ranks.  It is  `- (processor_props.comm_size-1)` which is `np.min(np.arange(3) - "maximal rank number") `